scripts/contentScript.js

```diff
@@ -212,6 +221,46 @@ async function showFloatingActionButton(orgName, orgRepo) {
 	}
 }
 
+// showing the important files in a pr
+async function showImpFileInPr(repoOwner,repoName,userId) {
+	const body = {
+		"repo_owner": repoOwner,
+		"repo_name": repoName,
+		"user_id": userId,
+		"is_github": true
+	}
+	let hashList = await apiCall(url,body);
+	// check the hashString and the last extension 
+	function checkString(hashString, lastExtension) {
+		let value = false;
+		hashList.forEach(item => {
+			let fileString = item.split('.');
+			let language = fileString.pop();
+			item = fileString.join(".");
+			if (hashString === item && lastExtension === language) {
+				return value = true;
```

Comment:
You are returning an assignment expression. This might work, but it is not very readable. Either only change the value to `true` or directly return `true` here.

Reply:
Removed function